app/src/main/java/com/github/se/travelpouch/ui/home/TravelList.kt

```diff
@@ -89,6 +91,29 @@ fun TravelListScreen(
               Icon(imageVector = Icons.Default.Add, contentDescription = "Add")
             }
       },
+      //      topBar = {
+      //          TopAppBar(
+      //              title = { Text("Travel") },
+      //              navigationIcon = {
+      //              IconButton(
+      //                  onClick = { navigationActions.navigateTo(Screen.TRAVEL_LIST) },
+      //                  modifier = Modifier.testTag("goBackButton")) {
+      //                    Icon(
+      //                        imageVector = Icons.AutoMirrored.Outlined.ArrowBack,
+      //                        contentDescription = "Back")
+      //                  }
+      //            },
+      //            actions = {
+      //                IconButton(
+      //                    onClick = { navigationActions.navigateTo(Screen.NOTIFICATION) },
+      //                    modifier = Modifier.testTag("settingsButton")
+      //                ) {
+      //                    Icon(imageVector = Icons.Default.Notifications, contentDescription =
+      // null)
+      //                }
+      //            }
+      //          )
+      //      },
```

Comment:
Consider simply removing the lines instead of commenting them

Reply:
I have done it